salt/modules/zypper.py

```diff
@@ -115,6 +115,11 @@ def info_installed(*names):
         t_nfo = dict()
         # Translate dpkg-specific keys to a common structure
         for key, value in pkg_nfo.items():
+            try:
+                value = value.encode('UTF-8', errors='replace')
```

Comment:
I would certainly ignore garbage symbols. They are inconvertable garbage that with the `replace` will turn into a convertable garbage. Still garbage, that will appear in the descriptions everywhere. Instead, using `ignore`, will simply yank them away, which is exactly what is needed.